src/fragments/lib/in-app-messaging/integrate-your-application/js/hoc-setup.mdx

```diff
@@ -0,0 +1,10 @@
+<!-- TODO in-app validate the library here. What should this be for things other than React-->
+```js
+import { withInAppMessaging } from '@aws-amplify/ui-react';
```

Comment:
This should include the css import as well (or this import should be covered somewhere else on this set of setup steps before the mostly complete example at the bottom).

Reply:
I think the idea here is to drop all the other stuff and focus on what we're teaching here - that there is a HOC and this is how to use it. I don't want to distract from that because the styling is not relevant to learning this concept.